README.md

```diff
@@ -218,15 +234,31 @@ and will not work if `{ style: 'compressed' }` is used in the third argument to
 
 ### Custom Importers
 
-If you use Webpack's tilde notation, like `@use '~accoutrement/sass/tools'`,
-you'll need to tell `runSass` how to handle that. That will require writing a
-[custom importer](https://sass-lang.com/documentation/js-api/interfaces/FileImporter)
-and passing it into the configuration for `runSass`:
+If you use the Dart Sass [Node.js package importer][pkg-importer] (e.g. `@use
+'pkg:accoutrement'`), you'll need to include `NodePackageImporter` in the
+options passed to `runSass`:
```

Comment:
Should True automatically inject the `NodePackageImporter`? That's the route I went for Herman. 🤔

Reply:
@jamesnw Can you think of any reason why adding `NodePackageImporter` by default might cause trouble? I'm leaning toward adding it, but allowing users to remove it with e.g. `importers: []` or something.

Reply:
Added a version of this in https://github.com/oddbird/true/pull/274/commits/b2c49ae31680e9d29e8bc027bb6890a8b9aaa9c7

Reply:
I can't think of any risks taking this approach, that makes sense to me.